roadmap-reviews/2020/review_3_june_20.md

```diff
@@ -0,0 +1,183 @@
+# End of ROADMAP 2020 review
+
+We've reached (and slightly shot past) the end of our 2020 roadmap, and it's
+time to review of our activities over the past year.
+
+## Cross-implementation interoperability
+
+- __in-toto-golang__ Apart from
+  [ed25519 signing](https://github.com/in-toto/in-toto-golang/pull/48) and
+  [bug fixes in the canonical JSON implementation](https://github.com/in-toto/in-toto-golang/pull/50)
+  discussed in the previous reviews, work began in earnest on bringing
+  in-toto's _runlib_ to the Go implementation. Christian Rebischke (@shibumi)
+  joined the in-toto team to implement this as part of Google Summer of Code
+  (GSoC). in-toto is participating in GSoC through the Cloud Native Computing
+  Foundation(CNCF). In the process, Christian also identified a nil pointer
+  dereference in Go's standard library for RSA, and he opened a
+  [pull request](https://go-review.googlesource.com/c/go/+/240008/) to address
+  it. Efforts are also underway to improve interoperability between this
+  implementation and the reference implementation.
```

Comment:
Not sure how much detail we want to provide, but @shibumi also added symlink recording support and works on exclude patterns:  https://github.com/in-toto/in-toto-golang/pulls?q=is%3Apr+author%3Ashibumi+